src/macaron/slsa_analyzer/package_registry/jfrog_maven_registry.py

```diff
@@ -132,6 +117,24 @@ def __init__(
         download_timeout: int | None = None,
         enabled: bool | None = None,
     ) -> None:
+        """
+        Instantiate a JFrogMavenRegistry object.
+
+        Parameters
+        ----------
+        hostname : str
+            The hostname of the JFrog instance.
+        repo : str
+            The Artifactory repository with Maven layout on the JFrog instance.
+        request_timeout : int
+            The timeout (in seconds) for regular requests made to the package registry.
+        download_timeout : int
+            The timeout (in seconds) for downloading files from the package registry.
+        enabled : bool
```

Comment:
I think it should be this instead (as these are the types of constructor params, not the types of instance variables): 

```suggestion
        hostname : str | None
            The hostname of the JFrog instance.
        repo : str | None
            The Artifactory repository with Maven layout on the JFrog instance.
        request_timeout : int | None
            The timeout (in seconds) for regular requests made to the package registry.
        download_timeout : int | None
            The timeout (in seconds) for downloading files from the package registry.
        enabled : bool | None
```

Reply:
See commit [33e0321](https://github.com/oracle/macaron/pull/436/commits/33e03210f8109f1c5992ff0035926c4b73309ad4)